samples/seh/Makefile

```diff
@@ -0,0 +1,5 @@
+XBE_TITLE = seh
```

Comment:
I think "seh" isn't a good sample name; it's not recognizable enough for noobs. "seh-exceptions" might be more suitable.

Reply:
It probably won't be part of the PR anyway, it's more of a demonstration of the current state. Imho it doesn't make for a good sample.

Reply:
While this particular code might not be a good sample (yet), I think we should still have a sample. SEH allows exceptions in  C (outside of C++), so this is kind of special and deserves a sample. However, the sample should probably be kept very simple, with a link to the docs for more information.